demo/mkldnn_quant/quant_aware/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+CMAKE_MINIMUM_REQUIRED(VERSION 3.2)
+
+project(mkldnn_quantaware_demo CXX C)
+set(DEMO_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR})
+set(DEMO_BINARY_DIR ${CMAKE_CURRENT_BINARY_DIR})
+
+option(USE_GPU      "Compile the inference code with the support CUDA GPU" ON)
+option(USE_PROFILER "Whether enable Paddle's profiler." OFF)
+
+set(USE_SHARED OFF)
+
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+if(NOT PADDLE_ROOT)
+  set(PADDLE_ROOT ${DEMO_SOURCE_DIR}/fluid_inference)
```

Comment:
Because there are two different ways to get paddle inference lib. build and make fluid_inference_install_dir is one way. Another way is download like [cpu_avx_mkl](https://www.paddlepaddle.org.cn/documentation/docs/zh/1.5/advanced_usage/deploy/inference/build_and_install_lib_cn.html#id1). I updated the document it should be more clear now.

Reply:
I think the `cpu_avx_mkl` does not necessarily support DNNL, does it?

Reply:
I checked, the cpu_avx_mkl link, seems support DNNL, and has newest version 1.7.1, but when I use that to build the application, I have this error `google::base::CheckOpMessageBuilder::NewString[abi:cxx11]()`. so now I only put in the doc building from source, not using pip etc.